packages/instrumentation-web-vitals/src/index.ts

```diff
@@ -10,17 +12,21 @@ const map = {
   ttfb: getTTFB,
 };
 
-const webVitalsInstrumentation: Instrumentation = async () => {
-  Object.entries(map).forEach(([indicator, executor]) => {
-    executor((metric) => {
-      agent.api.pushMeasurement({
-        type: 'web-vitals',
-        values: {
-          [indicator]: metric.value,
-        },
+const webVitalsInstrumentation: Instrumentation = {
+  initialize: async () => {
```

Comment:
```suggestion
  initialize: () => {
```

This does not need to be `async`